lib/PuppeteerSharp/LaunchOptions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace PuppeteerSharp
+{
+    public class LaunchOptions
+    {
+        public bool AppMode { get; set; }
+
+        public bool IgnoreHTTPSErrors { get; set; }
+
+        public bool IgnoreDefaultArgs { get; set; }
+
+        public bool Headless { get; set; } = true;
+
+        public string ExecutablePath { get; set; }
+
+        public int SlowMo { get; set; }
+
+        public string[] Args { get; set; } = Array.Empty<string>();
+
+        public bool HandleSIGINT { get; set; } = true;
+
+        public bool HandleSIGTERM { get; set; } = true;
+
+        public bool HandleSIGHUP { get; set; } = true;
+
+        public int Timeout { get; set; } = 30_000;
+
+        public bool Dumpio { get; set; }
```

Comment:
How about `DumpIO`?
See: [.NET example](https://github.com/aspnet/KestrelHttpServer/blob/3fc69dc71f309fe59be48e0d5cf8592c10bfd120/test/Kestrel.Core.Tests/HttpResponseStreamTests.cs#L109) And [Code analysis rule](https://msdn.microsoft.com/en-us/library/ms182256.aspx).